src/scheduleBot_Manager_TGAPI.py

```diff
@@ -0,0 +1,438 @@
+        group_response = await self.group_response(message, response, "full")
+
+        if group_response[0] is False:
+            response_message = group_response[1]
+
+        else:
+            response_message = await self.manager.full_response(
+                group=group_response[1]
+            )
+
+        await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+
+    async def current_lesson(self, message: aiogram.types.message):
+        response = await self.get_info(message)
+        group_response = await self.group_response(
+            message,
+            response,
+            "currentlesson"
+        )
+
+        if group_response[0] is False:
+            response_message = group_response[1]
+
+        else:
+            response_message = await self.manager.current_lesson_response(
+                group=group_response[1],
+                name=response[2]
+            )
+
+        await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+
+    async def next_lesson(self, message: aiogram.types.message):
+        response = await self.get_info(message)
+        group_response = await self.group_response(
+            message,
+            response,
+            "nextlesson"
+        )
+
+        if group_response[0] is False:
+            response_message = group_response[1]
+
+        else:
+            response_message = await self.manager.next_lesson_response(
+                group=group_response[1],
+                name=response[2]
+            )
+
+        await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+
+    async def timetable_mes(self, message: aiogram.types.message):
+        await self.get_info(message)
+        response_message = await self.manager.timetable_mes_response()
+        await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+
+    async def left(self, message: aiogram.types.message):
+        response = await self.get_info(message)
+        name = response[2]
+        response_message = await self.manager.left_response(name)
+        await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+
+    async def find_info(self, message: aiogram.types.message):
+        await self.get_info(message)
+        info_request = message.text[message.entities[0].length:].strip()
+        response_message = await self.manager.find_info_response(info_request)
+        send_res = await self.send_message(
+            message.chat.id,
+            response_message,
+            parse_mode='Markdown'
+        )
+        if send_res[0] is False and type(send_res[1]) \
+                == aiogram.utils.exceptions.MessageIsTooLong:
```

Comment:
Here you can do from `aiogram.utils.exceptions import MessageIsTooLong` and use just `MessageIsTooLong`